tests/cloud/pub_sub/mocks.py

```diff
@@ -0,0 +1,171 @@
+from octue.cloud.pub_sub import Subscription, Topic
+from octue.cloud.pub_sub.service import Service
+
+
+MESSAGES = {}
+
+
+class MockTopic(Topic):
+    """A mock topic that registers in a global dictionary rather than Google Pub/Sub.
+
+    :param str name:
+    :param str namespace:
+    :param MockService service:
+    :return None:
```

Comment:
??

Reply:
Ah I forgot to update them after I based them on another class!